components/ChatMessageDeleteModal.vue

```diff
@@ -0,0 +1,49 @@
+<template>
```

Comment:
We have a generic `ConfirmModal` component, which I've not told you about.  

I think you could use that and put the message in the default slot rather than introduce a new component.